doc/packet_struct_notation.md

```diff
@@ -17,31 +23,33 @@ All mentioned data types are assumed to be little-endian (least-
 significant byte first, least significant bit last) and of same size
 regardless of architecture.
 
-= Typical description of a packet =
+### Typical description of a packet
 
- /// Notifies the client about entering a chatroom (ZC_ENTER_ROOM).
- /// 00db <packet len>.W <chat id>.L { <role>.L <name>.24B }*
- /// role:
- ///     0 = owner (menu)
- ///     1 = normal
+```
+Notifies the client about entering a chatroom (ZC_ENTER_ROOM).  
+00db <packet len>.W <chat id>.L { <role>.L <name>.24B }*  
+role:  
+  0 = owner (menu)  
+  1 = normal  
+```
 
 The first line contains a brief description of what the packet does,
-or what it is good for, followed by it's AEGIS name in parentheses;
-first two letters of the AEGIS name specify origin (first letter)
+or what it is good for, followed by it's `AEGIS` name in parentheses;
```

Comment:
Why did we add quotes for a few words here?

Reply:
I thought that those names would look cooler once we generate the docs because the `backticks make them look different`

Reply:
<img width="335" alt="Screenshot 2023-03-30 at 12 35 41" src="https://user-images.githubusercontent.com/13068064/228823848-7826d408-744b-42bf-b765-4a64edb8532c.png">

https://github.com/Danil0v3s/rathena/blob/master/doc/packet_struct_notation.md